.github/workflows/auto-pr-to-release.yml

```diff
@@ -18,11 +18,100 @@ on:
         description: 'PR number to process'
```

Comment:
This is just an update to the auto-pr pipeline. Helps me now put this to 1.5.1 and 1.6.1